deployment/bicep/backend.bicep

```diff
@@ -0,0 +1,433 @@
+  //    protocolSettings: {
+  //      smb: {}
+  //    }
+    cors: {
+      corsRules: []
+    }
+    shareDeleteRetentionPolicy: {
+      enabled: false
+      days: 0
+    }
+  }
+}
+
+// Azure Queue Service for Azure Storage Account
+resource Microsoft_Storage_storageAccounts_queueServices_storageAccounts_saapimcsbackend_name_default 'Microsoft.Storage/storageAccounts/queueServices@2021-04-01' = {
+  parent: storageAccounts_saapimcsbackend_name_resource
+  name: 'default'
+  properties: {
+    cors: {
+      corsRules: []
+    }
+  }
+}
+
+// Azure Table Service for Azure Storage Account
+resource Microsoft_Storage_storageAccounts_tableServices_storageAccounts_saapimcsbackend_name_default 'Microsoft.Storage/storageAccounts/tableServices@2021-04-01' = {
+  parent: storageAccounts_saapimcsbackend_name_resource
+  name: 'default'
+  properties: {
+    cors: {
+      corsRules: []
+    }
+  }
+}
+
+// Azure Function App (Linux, .NET Core 3.1)
+resource sites_funcappAPIMCSBackendMicroServiceA_name_resource 'Microsoft.Web/sites@2018-11-01' = {
+  name: sites_funcappAPIMCSBackendMicroServiceA_name
+  location: 'West Europe'
+  kind: 'functionapp,linux'
+  properties: {
+    enabled: true
+    hostNameSslStates: [
+      {
+        name: 'funcappapimcsbackendmicroservicea.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Standard'
+      }
+      {
+        name: 'funcappapimcsbackendmicroservicea.scm.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Repository'
+      }
+    ]
+    serverFarmId: serverfarms_appsvcplanAPIMCSBackend_name_resource.id
+    reserved: true
+    isXenon: false
+    hyperV: false
+    siteConfig: {
+      numberOfWorkers: 1
+      linuxFxVersion: 'dotnet|3.1'
+      alwaysOn: true
+      http20Enabled: false
+    }
+    scmSiteAlsoStopped: false
+    clientAffinityEnabled: false
+    clientCertEnabled: false
+    hostNamesDisabled: false
+    containerSize: 1536
+    dailyMemoryTimeQuota: 0
+    httpsOnly: false
+    redundancyMode: 'None'
+  }
+}
+
+// Azure Function App (Container)
+resource sites_funcappAPIMCSBackendMicroServiceB_name_resource 'Microsoft.Web/sites@2018-11-01' = {
+  name: sites_funcappAPIMCSBackendMicroServiceB_name
+  location: 'West Europe'
+  kind: 'functionapp,linux,container'
+  properties: {
+    enabled: true
+    hostNameSslStates: [
+      {
+        name: 'funcappapimcsbackendmicroserviceb.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Standard'
+      }
+      {
+        name: 'funcappapimcsbackendmicroserviceb.scm.azurewebsites.net'
+        sslState: 'Disabled'
+        hostType: 'Repository'
+      }
+    ]
+    serverFarmId: serverfarms_appsvcplanAPIMCSBackend_name_resource.id
+    reserved: true
+    isXenon: false
+    hyperV: false
+    siteConfig: {
+      numberOfWorkers: 1
+      linuxFxVersion: 'DOCKER|mcr.microsoft.com/azure-functions/dotnet:3.0-appservice-quickstart'
```

Comment:
What do we want with DOCKER here?

Reply:
it is the descriptor of the tech stack: here the Azure Function App running from docker container (no code to deploy)